migrate-odin.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+set -e
```

Comment:
We need to remove `-e`, enabling this will exit the script on errors due to which 
```shell
if [ $? -ne 0 ]; then
```
becomes meaningless